content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,60 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define partial container spec as a custom profile in either YAML or JSON format, and apply it using the `--custom` flag.
+
+{{< note >}}
+- Custom profile only supports the modification of the debug container spec. It does not support the modification of the Pod spec of the debug target.
+- Modifications via custom profile is not allowed for certain fields such as command, image, lifecycle, volume devices and container name. In the future, more fields can be added to the disallowed list if required.
```

Comment:
How about:
```suggestion
Custom profiles only support modification of the ephemeral debug container. You cannot use
custom profiles to modify running containers.

Modifications via a custom profile are not allowed for certain fields, including: `command`, `image`, `lifecycle`, `name`, and fields that define access to storage.
```
?

Reply:
As [above](https://github.com/kubernetes/website/pull/48154#discussion_r1789653676), I think we should not limit to just an ephemeral container.
And last sentence in your suggestion is good for me, so I've addressed.